### ISSUE TITLE
A few JSONErrors

### ISSUE THREAD

**Issue by edouardpoitras** [jdreaver/oanda-rest-api#2]
I get the following error when pulling data using the `instruments` function:

```
Data: JSONError "Error in $.instruments[0].halted: failed to parse field halted: expected String, encountered Boolean"
```

Changing line 55 of Rates.hs from using `Maybe String` to `Maybe Bool` seems to do the trick.

But then I get:

```
Data: JSONError "Error in $.instruments[0].interestRate: failed to parse field interestRate: mempty"
```

Where changing line 56 of Rates.hs from using `Maybe Decimal` to `Maybe Object` makes it work again:

```
...instrumentInterestRate = Just (fromList [("USD",Object (fromList [("bid",Number 0.0),("ask",Number 4.0e-3)])),("EUR",Object (fromList [("bid",Number 0.0),("ask",Number 4.0e-3)]))])...
```

Although I don't know enough about Aeson (or Haskell for that matter) to know if using `Maybe Object` is the correct approach.

I'll submit a pull request.

**Comment by jdreaver** [comment]
Fixed with #3.

**Comment by jdreaver** [comment]
Fixed with #3.
